src/index.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+import ReactDOM from "react-dom";
```

Comment:
Oddly enough this is not seen as a file rename by git.
Also the quotes are still double here.